coredns/Acornfile

```diff
@@ -1,11 +1,14 @@
 args: {
-	// Number of replicas to run
+	// Number of replicas to run (example: --scale 1)
     scale: 1
 
-    // String of a Corefile to run with
+    // String of a Corefile to run with (example: --corefile @Corefile)
     corefile: ""
 
-    // Version of CoreDNS to run
+    // YAML file where each key is a file and its value is the content of that file. (example: --configs @config.yaml)
```

Comment:
```suggestion
    // YAML file where each key is a filename and its value is the content of that file. (example: --configs @config.yaml)
```

Reply:
Why go this route vs. secret? Would there ever be sensitive information in this file? If so, then a secret is probably the better route.